### ISSUE TITLE
Unable to import static method assertThat

### ISSUE THREAD

**Issue by plackemacher** [square/assertj-android#129]
I replaced the line in my `build.gradle` file that declared using Fest-Android with one for AssertJ-Android:

``` groovy
-    androidTestCompile('com.squareup:fest-android:1.0.+') {
+    androidTestCompile('com.squareup.assertj:assertj-android:1.0.0') {
```

The issue is that after replacing all static imports of `import static org.fest.assertions.api.ANDROID.assertThat;` with `import static org.assertj.android.api.Assertions.assertThat;`, the static import isn't found therefore my Robolectric tests fail to compile. If I change the Gradle line to use `compile` instead of `androidTestCompile`, the tests work correctly but obviously that's not optimal nor what the documentation describes.

The error returned by javac is the following:

```
File.java:∞ error: package org.assertj.android.api does not exist
import static org.assertj.android.api.Assertions.assertThat;
                                     ^
```

**Comment by pboos** [comment]
@JakeWharton Your comment "Future-proofing. Maybe we want to ship custom lint rules in the future." is not true anymore. .aar dependencies for androidTestCompile is not allowed since 1.1 android gradle plugin. ;-) so actually it was un-future-proofing ;-).

**Comment by plackemacher** [comment]
@pboos could you post a reference to where that is stated? I'm a bit curious.

**Comment by pboos** [comment]
@plackemacher @JakeWharton i guess i remembered wrong. after checking again i see now that it was because i as well added assertj-android as provided. That I do because android studio doesn't seem to pick up the androidTestCompile dependencies correctly. Guess will have to find another way to get that working. Or any idea? ;-)

**Comment by jhansche** [comment]
@pboos `androidTestCompile` works for me, and Android Studio picks it up and allows me to use those classes in the test source.  I also had to add this, because my robolectric test code is in `src/test/java`:

```gradle
sourceSets {
    androidTest.setRoot('src/test')
}
```

There was, however, a period of time where Android Studio stopped recognizing my test dependencies and source files, so I couldn't edit any of the src/test/java code as a full-on "class" (just as a .java file).  I fixed that by simply re-importing my project into Android Studio -- then it started recognizing my test classes and dependencies again.
